factory/django.py

```diff
@@ -152,7 +164,11 @@ def _get_or_create(cls, model_class, *args, **kwargs):
             key_fields[field] = kwargs.pop(field)
         key_fields['defaults'] = kwargs
 
+        related_kwargs = cls._extract_related_kwargs(model_class, kwargs)
+
         instance, _created = manager.get_or_create(*args, **key_fields)
+        for key, data in related_kwargs.items():
+            getattr(instance, key).set(data)
```

Comment:
The logic to defer related kwargs assignment is present in both `_get_or_create` and `_create`. What do you think about creating a helper method that handles the related kwargs and chooses between Django's `create` and `get_or_create`?